src/modules/0.0.21/aws_codebuild/sql/after_install.sql

```diff
@@ -0,0 +1,21 @@
+CREATE
+OR REPLACE FUNCTION generate_put_ecr_image_build_spec (region TEXT, image_tag TEXT, repo_name TEXT, repo_uri TEXT, build_path TEXT) RETURNS TEXT LANGUAGE plpgsql AS $$
+BEGIN
+  RETURN 'version: 0.2
+
+  phases:
+    pre_build:
+      commands:
+        - echo Logging in to Amazon ECR...
+        - aws ecr get-login-password --region ' || region || ' | docker login --username AWS --password-stdin ' || repo_uri || '
```

Comment:
I see now that codebuild has no explicit support for inserting into ECR, but it's a pattern you can configure it to follow like in other CI systems, and that's why you can't (easily) turn it into a single `apply` call. I kinda *do* want a `codebuild_simplified` module, now, since github -> codebuild -> ecr and then ecr -> ecs feels like a very common, desired pipeline.